tests/test_marquez_dag.py

```diff
@@ -440,7 +437,7 @@ def test_marquez_dag_with_extractor(mock_get_or_create_marquez_client,
             source_name='dummy_source_name',
             namespace_name=DAG_NAMESPACE,
             fields=[],
-            run_id=run_id
+            run_id=None
```

Comment:
_Minor_: Since `run_id` is optional, maybe just omit the parameter from the method call (i.e. the client defaults to none if not provided)?